benchmarks/run.py

```
@@ -0,0 +1,340 @@
+    except json.JSONDecodeError as exc:
+        return InvocationResult(
+            invocation=inv, status="error", elapsed_sec=elapsed,
+            output=None, error_message=f"output JSON parse failed: {exc}",
+        )
+    return InvocationResult(
+        invocation=inv, status="ok", elapsed_sec=elapsed, output=parsed,
+    )
+
+
+def _default_runner(cmd: list[str], out_path: Path) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)